patent.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import requests
+import json
+
+
+def patent(**kwargs):
+    patent_url = 'https://api.nasa.gov/patents/content'
```

Comment:
I would remove this static URL and instead make it a parameter to this function. We'll want to provide the URL for this API via a configuration file, most likely. We generally want to stay away from statically declaring things in a function.